nan_maybe_43_inl.h

```diff
@@ -217,7 +217,21 @@ NAN_INLINE Maybe<int> GetEndColumn(v8::Local<v8::Message> msg) {
 NAN_INLINE MaybeLocal<v8::Object> CloneElementAt(
     v8::Local<v8::Array> array
   , uint32_t index) {
-  return array->CloneElementAt(GetCurrentContext(), index);
+  v8::Local<v8::Context> context = GetCurrentContext();
+#if (NODE_MODULE_VERSION >= NODE_6_0_MODULE_VERSION)
+  v8::EscapableHandleScope handle_scope(v8::Isolate::GetCurrent());
```

Comment:
The context handle leaks to the surrounding scope, since it is created before the scope.